sync-tool/bin/update

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env php
+<?php
+
+require_once __DIR__ . '/../vendor/autoload.php';
+require_once __DIR__ . '/../UpdateFromPerlSrc.php';
```

Comment:
this requirement should be removed, relyign on the composer autoloader instead (actually using ``autoload-dev`` for that as the sync-tool only needs to be autoloadable when working on the library, not when using it)